Makefile

```diff
@@ -56,18 +56,14 @@ build_compiled: ## build production container image
 		-t $(COMPILED_IMAGE) \
 		--target $(COMPILE_TARGET) .
 
-build_serialize:
+build_debug: ## build container image for debugging
+	COMPILED_TAG_NAME=$(COMPILED_TAG_NAME)-debug COMPILE_OPTION="REPRO=\\\nDEBUG=Y" $(MAKE) build
```

Comment:
Oh nice, now can have multiple debug images if needed.

Reply:
Yes. I guess this was mostly about making the naming of images consistent across default, debug and serialize.